stories/Card.stories.js

```diff
@@ -45,13 +44,11 @@ export const card = () => (
 );
 
 export const cardWithLinks = () => (
-  <Router>
-    <Card
-      title="Title"
-      subtitle={<a to="/#">APOE_HUMAN - P02649</a>}
-      links={links}
-    >
-      {getLipsumSentences()}
-    </Card>
-  </Router>
+  <Card
+    title="Title"
+    subtitle={<a to="/#">APOE_HUMAN - P02649</a>}
```

Comment:
Shouldn't this be `<Link to="">`, rather `<a to="">`? I think Anchor tag should only be used with `href` attribute.